addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -2084,7 +2084,7 @@ export function rgbToHex(rgb = '') {
 
 export function getRangePosition(el, document, options = {}) {
     const selection = document.getSelection();
-    if (!selection.isCollapsed || !selection.rangeCount) return;
+    if (!selection.rangeCount) return;
```

Comment:
I don't remember exactly, but it's probably similar than the t-out situation : 
A bug that was not very visible before the addition of dynamic placeholder, and since I encounter it during this task development, I fixed it.

Reply:
Same then, please isolate each separate fix into a separate commit.